### PR TITLE
fix(build): unify docker image UTC timezone

### PR DESCRIPTION
**Pull request by zexi**
**What this PR does / why we need it**:

**Does this PR need to be backport to the previous release branch?**:

- release/3.8

/cc @ioito @swordqiu 
/area utils

### CODE REVIEW

**Comment by zexi** [comment]
/area util
Ref: https://github.com/yunionio/cloudpods/issues/12801

File: build/docker/Dockerfile.host [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM registry.cn-beijing.aliyuncs.com/yunionio/host-base:v0.4.0
 
 MAINTAINER "Yaoqi Wan wanyaoqi@yunionyun.com"
 
-ENV TZ Asia/Shanghai
+ENV TZ UTC
```
Comment (swordqiu) [inline build/docker/Dockerfile.host:5]: RUN cp /usr/share/zoneinfo/UTC /etc/localtime

File: build/docker/Dockerfile.vpcagent [hunk 2]
```diff
@@ -2,5 +2,6 @@ FROM registry.cn-beijing.aliyuncs.com/yunionio/openvswitch:2.10.5-1
 
 MAINTAINER "Yousong Zhou <zhouyousong@yunion.cn>"
 
+ENV TZ UTC
```
Comment (swordqiu) [inline build/docker/Dockerfile.vpcagent:5]: RUN cp /usr/share/zoneinfo/UTC /etc/localtime

File: build/docker/Dockerfile.webconsole-base [hunk 3]
```diff
@@ -2,11 +2,9 @@ FROM registry.cn-beijing.aliyuncs.com/yunionio/kubectl:1.18.6
 
 MAINTAINER "Zexi Li <lizexi@yunionyun.com>"
 
-ENV TZ Asia/Shanghai
+ENV TZ UTC
```
Comment (swordqiu) [inline build/docker/Dockerfile.webconsole-base:5]: RUN cp /usr/share/zoneinfo/UTC /etc/localtime

Comment (zexi) [reply build/docker/Dockerfile.webconsole-base:5]: 这些没有 cp 的是镜像里面没有这个 UTC 文件，测试设置环境变量 TZ=UTC 也能生效

**Pull request merged**